projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -751,6 +757,17 @@ public function update_settings() {
 					}
 					break;
 
+				case 'woocommerce_store_address':
+				case 'woocommerce_store_address_2':
+				case 'woocommerce_store_city':
+				case 'woocommerce_default_country':
+				case 'woocommerce_store_postcode':
+					$sanitized_value = sanitize_text_field( $value );
+					if ( update_option( $key, $sanitized_value ) ) {
+						$updated[ $key ] = $sanitized_value;
+					}
+					break;
+
```

Comment:
It looks like `woocommerce_onboarding_profile` needs some sanitization still? Would this work?

```php
$value = array_map( 'sanitize_text_field', (array) $value );
```